packages/@aws-cdk/aws-ecs/test/cluster.test.ts

```diff
@@ -2141,6 +2141,54 @@ describe('cluster', () => {
 
   });
 
+  test('can add ASG capacity via Capacity Provider with default capacity provider', () => {
```

Comment:
Let's also either add a new integ test or modify an existing autoscaling integ test to set default capacify provider strategy.